src/plugins/dna_export/src/ExportTasks.h

```diff
@@ -90,6 +97,8 @@ class ExportMSA2MSATask : public DocumentProviderTask {
     QString url;
     QString format;
     QList<DNATranslation *> aminoTranslations;    // amino translation for a sequences in alignment. If not NULL -> sequence is translated
+    bool trimGaps;                                // trim gaps before translation of not
```

Comment:
nit: if something is not expected to be changed inside of the class it can be declared as a 'const'

Reply:
Done